squire_lib/src/tournament.rs

```diff
@@ -961,6 +961,116 @@ impl Tournament {
 }
 
 impl TournamentSeed {
+    const MAGIC_SYNONYMS: &[&'static str] = &[
```

Comment:
I know you pulled these from the desktop (which were pulled from a thesaurus page), but feel free to make additions to them if you'd like.